### ISSUE TITLE
How to get flow.flo

### ISSUE THREAD

**Issue by HrsPythonix** [sniklaus/softmax-splatting#2]
Could you tell me specifically how I can get the flow.flo in your repo?

**Comment by sniklaus** [comment]
You can use any optical flow estimator that you find suitable for your task, for example the ones below.

https://github.com/sniklaus/pytorch-spynet
https://github.com/sniklaus/pytorch-pwc
https://github.com/sniklaus/pytorch-liteflownet
https://github.com/sniklaus/pytorch-unflow

**Comment by kwea123** [comment]
More generally, you can use https://github.com/liruoteng/OpticalFlowToolkit/blob/master/lib/flowlib.py
that handles many more flow file operations.
